.eslintrc

```diff
@@ -35,7 +35,7 @@
     ],
     "max-len": [
       2,
-      80,
+      120,
```

Comment:
What's this? Line width?

Reply:
Yes. I changed this as it would otherwise make a lot of the code look really ugly with unnecessary line breaks.

I believe we should remove all the local rules here and use JavaScript standard or Google code-style directly, but that would be a different PR.